web/src/i18n/locales/en/activities/overview.yaml

```
@@ -12,8 +12,8 @@ summary:
 detail:
   # Standard is for HIT and MMIS
   standard:
-    ? subheader
-
+    helpText: >-
+      Include as much detail necessary to explain the activity.
```

```suggestion
      Include as much detail as is necessary to explain the activity.
```
`as is` creates a complete sentence.

This language is fine if state users are going to know what it means to explain the activity and I'll approve it regardless because this change is good! But one note to consider: we're asking to distinguish between summarizing in the first text box and explaining in this one. Is that distinction going to be clear enough, or do we need to offer more about what they're explaining?

---

@jameshupp If you have some suggestions for changing it, maybe we can bake it into #1027?